configs/components/oasis3mct/oasis3mct.env.yaml

```diff
@@ -1,5 +1,9 @@
-environment_changes:
+add_compiletime_environment_changes:
         choose_computer.fc:
                 ftn:
                         add_export_vars:
                                 - 'OASIS_FFLAGS=-emf'
+        choose_computer.name:
+                levante:
+                        add_export_vars:
+                                OASIS_FFLAGS: '"-march=core-avx2 -mtune=core-avx2"'
```

Comment:
These flags are then passed to `cmake` somehow, right?

Reply:
Some versions of OASIS have that on their cmake files, others not. This was written here for the one from AWICM3 which can handle this. I am not sure about the one for AWIESM2